exercises/practice/list-ops/list-ops.wren

```diff
@@ -0,0 +1,62 @@
+class ListOps {
+  construct new(elements) {
+    Fiber.abort("Remove this statement and implement this class")
+  }
+
+  // return the items as a List
+  toList {
+    Fiber.abort("Remove this statement and implement this method")
+  }
+
```

Comment:
Filling out the stub file with the expected methods.

Reply:
If we could improve Wren's handling of missing functions perhaps we coudl avoid this in the future... wonder if this is perhaps worth an issue filed against Wren proper?

Reply:
My first commit omitted them (a class goes here: write it). But your (albeit temporary)  confusion about the method names was bound to be repeated by students, so I thought it prudent to add them.

Reply:
All wren exercises have always had them (with correct signatures) because otherwise it's a _compile time_ error - and the test suite can't even start, etc... so the testing tooling would need to get more complex just to be able to generate anything other than a complete hard failure (which looks like an infrastructure issue).

Reply:
What about this?
![image](https://user-images.githubusercontent.com/122470/180228000-c878e767-2075-4a04-82ef-b884a1166745.png)

Reply:
Even if the wren file is empty
![image](https://user-images.githubusercontent.com/122470/180228332-86383887-e6b5-43cd-b690-e075bd8b0128.png)

Reply:
The first example (just new) only works [well] in this case because ALL tests start with a call to the `new` method... if there were other static methods and those did not exist... then they would need to exist or you get a hard error.

The second example - the hard error (Wren Console can't compile the script or test suite)... I don't think is super welcoming for students.  Would/should it be reasonable for harder exercises?  I dunno...  I was trying VERY hard not to have any "boiler plate comments" which students never remove and drive me nuts.

If we went with something like this:

```
class HighScores {
}
```

You'd get a soft runtime error of:

```
HighScores metaclass does not implement 'new(_)'.
at test(_,_) block argument (./high-scores.spec.wren line 7)
```